### PR TITLE
[cli] Install will create eslintrc

### LINKED ISSUE #779: Improve flow-typed and eslint compatibility
After installation of flow-typed in a project in the `/flow-typed` folder there are often style conflicts with .eslintrc rules at the root of the project.
It is normal because the definitions are defined with a style that comes from the github repository.

There are many possible solutions : 
* Ask ESLint to ignore 'flow-typed' folder by default
* `flow-typed install` can configure `.eslintignore` to ignore flow-typed folder
* `flow-typed install` can create a `.eslintrc` in the flow-typed directory that disable all rules (root=true and empty rules)

### PR DESCRIPTION
**Pull request by Brianzchen**
Closes #779, #929

When starting a new project `flow-typed install` should be seamless and with zero config. With eslint it's not really the case and at least for me I'd add `flow-typed` into `.eslintignore`. There's basically no reason to run eslint rules at all in libdefs coming from `flow-typed` so this change adds an automatic `.eslintrc.js` that disables rules.

It can be overridden though for some unique use cases.

### CODE REVIEW

File: cli/src/lib/fileUtils.js [hunk 1]
```diff
@@ -18,6 +18,18 @@ export function copyDir(srcPath: string, destPath: string): Promise<void> {
   });
 }
 
+export function writeFile(destPath: string, content: string): Promise<void> {
```
Comment (pascalduez) [inline cli/src/lib/fileUtils.js:21]: We should probably start using Node's `fs/promises` at some point.

**Comment by pascalduez** [comment]
IMO this is going a bit too far, it's not `flow-typed` responsibility.   
Also:
-  in the current state it will add the ESLint config file even if the project isn't using ESLint.
- enforce the `.eslintrc.js` config file type whereas others exists
- enforce setting that kind of ignore/config into the `flow-typed` dir, whereas users might prefer dealing with this at the project root

**Comment by Brianzchen** [comment]
Good points... Let me come back to this

**Comment by GAntoine** [comment]
I agree with pascal here. Is this something we can add to the docs, with specified steps on how to make flow-typed work with eslint?

**Pull request closed without merging**